cli/app.go

```diff
@@ -18,6 +18,7 @@ const (
 	machineFlag      = "machine"
 	aliasRobotFlag   = "robot"
 	partFlag         = "part"
+	robotFqdnFlag    = "fqdn"
```

Comment:
Hopefully you won't need this nor `loginFlagKeyIDVal` if you rely on existing auth mechanisms as @zaporter-work suggested.

Reply:
irl feedback from benji: we are leaning towards using the main part id to identify the robot. (reason for this: it's what's available from viam.json).

there is some confusion here though; it's hard to get that id from the web UI. (the URL has robot id; you can get part id by going to 'debug configuration' in the ellipsis menu for the main part)

@zaporter-work we decided to punt to you -- do you have advice on 1) which ID to use for reload / restart and 2) how to communicate to users where to find it?

Reply:
> main part id to identify the robot

I suspect you want to identify the robot part instead of the robot. (because you want to be able to distinguish between restarting a module on the main part vs a subpart). Please correct me if you have a reason for finding the robot instead of the part

To find the part ID, a user can use this "Part ID" button to copy the id to their clipboard:
![20240418_17h07m13s_grim](https://github.com/viamrobotics/rdk/assets/121693134/45d5b326-abd3-4384-89a9-93d89ca5e9f7)

Reply:
We have named that dropdown the `Part status dropdown`

Reply:
ah this is the 'Live / Offline' pill next to the breadcrumbs on the secondary navbar -- awesome, that works

Reply:
using part Id with UI instructions in 8e186a4e. using `Part.Fqdn` from part ID in 7dffefae5 (rather than requiring user to pass in fqdn)

is Fqdn right? Or will I need LocalFqdn sometimes

Reply:
> Or will I need LocalFqdn sometimes

Passing the gauntlet back to @benjirewis 

I think using the Fqdn all the time will work (though perhaps LocalFqdn can improve connection performance?)

Reply:
I think you should use the `FQDN` and not the `LocalFQDN`. The local FQDN would, I think, only be usable if you're running the CLI on the same subnet as the machine. I realize we're probably expecting CLI users to run `viam module restart` from the same machine as the running RDK (in order to actually rebuild the module?), but we probably want to allow `viam module restart --restart-only` from a CLI user not running on the same machine. As far as performance, both Cheuk and I think the difference between using the regular and local FQDN would be fairly negligible in this case.